aimmo-game/simulation/world_map.py

```diff
@@ -89,10 +89,28 @@ class WorldMap(object):
     """
 
     def __init__(self, grid):
-        self.grid = grid
+        self._grid = grid
+
+    @classmethod
+    def _min_max_from_dimensions(cls, height, width):
+        max_x = int(math.floor(width / 2))
+        min_x = -(width - max_x - 1)
+        max_y = int(math.floor(height / 2))
```

Comment:
there are other /2 elsewhere in your PR if you wish to be consistent with the use of floor :)